packages/dso-toolkit/src/components/annotation/annotation.args.ts

```diff
@@ -1,65 +1,211 @@
 import { ArgTypes } from "@storybook/types";
-import { HandlerFunction } from "@storybook/addon-actions";
 
-import { noControl } from "../../storybook/index.js";
+import { Annotation, AnnotationActiveChangeEvent, AnnotationDiff, AnnotationWijzigactie } from "./annotation.models.js";
 
-import { Annotation } from "./annotation.models.js";
+/**
+ * Annotation Base
+ */
 
-export interface AnnotationArgs {
-  identifier: string;
-  open: boolean;
-  prefix: string;
-  dsoClose: HandlerFunction;
-  dsoClick: HandlerFunction;
+interface AnnotationArgsBase {
```

Comment:
niet AnnotationBaseArgs?

De const op r.17 is wel `annotationBaseArgs`

Reply:
Hehe, ik dacht dat ik een beslissing had genomen 🤡 

Ik denk dat we inderdaad voor `AnnotationBaseArgs` gaan :+1:

Reply:
Done!